.github/workflows/matplotlib-tests.yml

```
@@ -0,0 +1,69 @@
+name: matplotlib-tests
+
+on:
+  pull_request:
+    paths:
+      - tests/visualization_tests/**
+      - .github/workflows/matplotlib-tests.yml
+      - optuna/visualization/**.py
```

Same ordering as the optuna repo would be better.

```suggestion
      - .github/workflows/matplotlib-tests.yml
      - optuna/visualization/**.py
      - tests/visualization_tests/**
```